main.go

```diff
@@ -102,7 +102,7 @@ func main() {
 	}
 	reconciler.DNSProvider = f.Provider()
 	log.Info().Msgf("provider: %s", reconciler.DNSProvider)
-	log.Info().Msg("starting metrics")
+	log.Info().Msgf("starting metrics on %s", reconciler.Config.MetricsAddress)
```

Comment:
Consider making this change as separate PR, as it's touching the controller code and not being exactly the scope of this PR.